schemas/post-artifact-request.yml

```diff
@@ -3,6 +3,49 @@ title:              "Post Artifact Request"
 description: "Request a authorization to put and artifact or posting of a URL as an artifact. Note that the `storageType` property is referenced in the response as well."
 type:               object
 oneOf:
+  - title:          "Blob Artifact Request"
+    description: |
+      Request for to upload a generic blob artifact.
+    type:           object
+    properties:
+      storageType:
+        description: |
+          Artifact storage type, in this case `'blob'`
+        type:       string
+        enum:
+          - blob
+      contentType:
+        description: |
+          Artifact mime-type, when uploading artifact to the signed
+          `PUT` URL returned from this request this must given with the
+           `ContentType` header. Please, provide correct mime-type,
+           this make tooling a lot easier, specifically,
+           always using `application/json` for JSON artifacts.
+        type:       string
+        maxLength:  255
+      sha256:       {type: string, pattern: '^[0-9a-f]{64}$'}
+      md5:          {type: string, pattern: '^[0-9a-f]{32}$'}
```

Comment:
Okay, I agree... 

Randomly, should we consider adding another algorithm while we're at it?
I just saw this on review of hash algorithms for git by some crypto experts:
https://public-inbox.org/git/91a34c5b-7844-3db2-cf29-411df5bcf886@noekeon.org/

I'm quite happy with sha2-256, but adding SHAKE128 or PH128 on top of that might not be entirely crazy. Just if we are supposed to think forward a few years.